components/logging.js

```diff
@@ -43,8 +43,23 @@ const interestingLogs = [
  * Uses cloud logging APIs to list log entries from the
  * fleet engine resource matching the specified label.
  */
-async function fetchLogs(label, labelValues, daysAgo = 2, extra = "") {
+async function fetchLogs(
+  label,
+  labelValues,
+  daysAgo = 2,
+  extra = "",
+  lmfsMrOverride = "DeliveryFleet"
+) {
   const endPoint = "fleetengine.googleapis.com";
+  const odrdMonitoredResource = "fleetengine.googleapis.com/Fleet";
+  const lmfsMonitoredResource = `fleetengine.googleapis.com/${lmfsMrOverride}`;
```

Comment:
This should just be another constant, instead of an override, because LMFS and ODRD are two products.

Reply:
Done.